vips/image_test.go

```diff
@@ -242,11 +262,36 @@ func TestImageRef_RemoveOrientation__NoEXIF(t *testing.T) {
 	require.NoError(t, err)
 	defer image.Close()
 
-	image.RemoveOrientation()
+	err = image.RemoveOrientation()
+	require.NoError(t, err)
+
+	assert.Equal(t, 0, image.GetOrientation())
+}
+
+func TestImageRef_RemoveMetadata(t *testing.T) {
+	Startup(nil)
+
+	image, err := NewImageFromFile(resources + "jpg-24bit-icc-adobe-rgb.jpg")
+	require.NoError(t, err)
+	defer image.Close()
 
-	o := image.GetOrientation()
+	err = image.RemoveMetadata()
+	require.NoError(t, err)
+
+	assert.True(t, image.HasICCProfile())
```

Comment:
i would also check that metadata was removed

Reply:
done